gravitee-apim-console-webui/src/services-ngx/debug-api.service.spec.ts

```diff
@@ -75,7 +77,7 @@ describe('DebugApiService', () => {
         body: '{"key": "value"}',
       };
 
-      const api = {
+      const api: any = {
```

Comment:
why not an Api type ?

Reply:
test linked to your comment upper. i think is a legacy stuff to remove but 🤷‍♂️ :not-sure: wee can see it in dedicated PR no ?

Reply:
i love this emoji 🤷‍♂️